docs/module_docs/filter_isotopegrouper/isotope_grouper.md

```diff
@@ -0,0 +1,27 @@
+# Isotope grouper (filter)
+
+The **Isotope grouper** is found under **Feature list methods → Isotopes → Isotope grouper (filter).**
+
+![](isotope_grouper.png)
+
+
+##### Name suffix
+Suffix to be added to feature list name.
+
+##### m/z tolerance
+Maximum allowed difference between two m/z values in order for them to be considered the same. The value is specified both as absolute tolerance (in m/z) and relative tolerance (in ppm). The tolerance range is calculated using maximum of the absolute and relative tolerances.
+
+##### Retention time tolerance
+Maximum allowed difference between two retention time values.
+
+##### Mobility tolerance
+If enabled(and mobility dimension was recorded), isotopic peaks will only be grouped if they fit the given tolerance.
+
+##### Monotonic shape
+If true, then monotonically decreasing height of isotope pattern is required.
+
+##### Maximum charge
+Maximum charge to consider for detecting the isotope patterns.

```

Comment:
The charge state with the maximum number of detected isotope features will be used.